extensions/sql-migration/src/wizard/integrationRuntimePage.ts

```diff
@@ -510,22 +525,33 @@ export class IntergrationRuntimePage extends MigrationWizardPage {
 				CSSStyles: { ...styles.BODY_CSS }
 			}).component();
 
-		const instructions = utils.createRegistrationInstructions(this._view, false);
-
-		this._authKeyTable = utils.createAuthenticationKeyTable(this._view, '50px', '500px');
+		const refreshAndConfigureIRcontainer = this._view.modelBuilder.toolbarContainer()
+			.withToolbarItems(
+				[
+					{ component: this._refreshButton },
+					{ component: this._configureIRButton }
+				]
+			).component();
 
-		statusContainer.addItems([
-			this._dmsStatusInfoBox,
-			instructions,
-			this._authKeyTable]);
+		this._integrationRuntimeTable = utils.createIntegrationRuntimeTable(this._view);
 
 		container.addItems([
 			connectionLabelContainer,
-			statusContainer]);
+			this._dmsStatusInfoBox,
+			refreshAndConfigureIRcontainer,
+			this._integrationRuntimeTable
+			//statusContainer
```

Comment:
why we have this statusContainer commented? Is it intended to be removed?

Reply:
Yes, will be removed in the next PR